client/my-sites/sidebar/sidebar.jsx

```diff
@@ -447,7 +447,7 @@ export class MySitesSidebar extends Component {
 				link={ siteSettingsLink }
 				onNavigate={ this.onNavigate }
 				icon="cog"
-				preloadSectionName="settings"
+				preloadSectionName="settings-general"
```

Comment:
Did a quick test—I see `settings-general.js` load when I hover, then `settings.js` loads when I click. So we could have a slight regression here unless we can preload two sections.